identity/validator_queue_test.go

```diff
@@ -0,0 +1,100 @@
+package identity
+
+import (
+	"os"
+	"testing"
+
+	"github.com/Oneledger/protocol/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+var vq1 = utils.NewQueued([]byte("a"), 200, 1)
+var vq2 = utils.NewQueued([]byte("b"), 1000, 2)
+var vq3 = utils.NewQueued([]byte("c"), 300, 3)
+var vq4 = utils.NewQueued([]byte("d"), 2000, 4)
+
+var Vq *ValidatorQueue
+
+func setup() {
```

Comment:
Only use setup and teardown for side-effects (e.g. file I/O, database setup, etc.). Logs don't do anything.